controlnext/templates/controlnext/selectbasin.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'controlnext/basin_detail.html' %}
+{% load i18n %}
+
+{% block content %}
+<div class="container" style="height:150px">
+</div>
+<div class="container">
```

Comment:
Ook in deze file de indentatie netjes op 2 spaties. En hele bestand indenteren inclusief de blocks.